src/pages/index.js

```diff
@@ -116,3 +211,70 @@ const addCardImageModal = new PopupWithForm(
   handleAddCardSubmit
 );
 addCardImageModal.setEventListeners();
+
+const modalPreviewImage = new PopupWithImage("#image-modal");
+modalPreviewImage.setEventListeners();
+
+/* -------------------------------------------------------------------------- */
+/*                           Button Event Listeners                           */
+/* -------------------------------------------------------------------------- */
+
+profileEditButton.addEventListener("click", () => {
+  const user = userInfo.getUserInfo();
+  profileTitleInput.value = user.name;
+  profileDescriptionInput.value = user.description;
+  modalEditForm.open();
+});
+
+addNewCardButton.addEventListener("click", () => {
+  addFormValidator._disableSubmitButton();
+  addCardImageModal.open();
+});
+
+editProfileImageButton.addEventListener("click", () => {
+  editProfilePicFormValidator._disableSubmitButton();
+  editProfilePicture.open();
+});
+
+/* -------------------------------------------------------------------------- */
+/*                            Card Functionalities                            */
+/* -------------------------------------------------------------------------- */
+
+function createCard(cardData) {
+  const card = new Card(
+    cardData,
+    "#card-template",
+    handleImageClick,
+    handleRemoveCard,
+    handleCardLike
+  );
+
+  return card.getView();
+}
+
+function renderCard(cardData) {
+  const cardEl = createCard(cardData);
+
+  section.addItem(cardEl);
+}
+
+let section;
+Promise.all([api.loadInitialProfileData(), api.getInitialCards()])
+  .then(([data, initialCards]) => {
+    userInfo.setUserInfo(data);
```

Comment:
Here you need to set the avatar as well

Reply:
Hello - thank you for all of the great feedback. A quick question about this one. On the following line I have: userInfo.setUserProfilePicture(data.avatar); 
Is this not sufficient for setting the Avatar or do I need to do something else?

Reply:
> Is this not sufficient for setting the Avatar or do I need to do something else?

It's enough to set the avatar on start